pages/docs/modules/exec.mdx

```diff
@@ -0,0 +1,355 @@
+# exec
+
+## Functions
+
+### command
+
+```go filename="Function signature"
+command(file string, args ...string) exec.command
+```
```

Comment:
This is the first occurrence of variadic parameters in the documentation. Is this syntax OK?

Reply:
Yeah, I struggled with questions like this while working on the docs too.

And yes, I agree with this choice for variadic params.